githubapp/githubapp.go

```diff
@@ -203,16 +211,35 @@ func (g *GitHubApp) generateAppJWT() ([]byte, error) {
 // access token for this application installation with the requested
 // permissions and repositories.
 func (g *GitHubApp) AccessToken(ctx context.Context, request *TokenRequest) (string, error) {
-	appJWT, err := g.AppToken()
```

Comment:
This looks good, but should we return an error if request.Repositories is empty? We can say "use "AccessTokenAllRepos instead" or something.

Reply:
I think yes, if nil throw error, if empty thats ok, meaning no permissions. I will update.